hw/ip/i2c/data/BUILD

```diff
@@ -12,3 +12,8 @@ autogen_hjson_header(
         "i2c.hjson",
     ],
 )
+
+filegroup(
+    name = "all_files",
+    srcs = glob(["**"]),
```

Comment:
For this (and other all_files targets with subdirectories with their own BUILD files), would it make sense to include the subdirectories here instead of in flattened lists like the one that appears in hw/BUILD?

Reply:
I should have put this comment on hw/ip/i2c/BUILD, which could probably point to this filegroup in its own all_files target so that consumers of it don't need to know whether this particular BUILD file exists and have to point to it.

Reply:
I've restructured the `all_files` targets into a hierarchical form.